### ISSUE TITLE
julec: GCC give errors for transpiled Jule code

### ISSUE THREAD

**Issue by ghost** [the-xlang/x#1]
Hİ Mertcan! 
Well done, Jule is very nice programming language :) It's just wrong like this:

```
main() {
    const a int32 = 5;
    var b int32 = 25;
    outln(div(b, a));
}

div(x float64, y float64) float64 {
  ret x / y;
}
```

As a result, g++ reports incorrect code:

> 
> x.cxx: In function 'void _main()':
> x.cxx:205:15: error: '_b_a' was not declared in this scope; did you mean '_b'?
>   205 |   _outln(_div(_b_a));
>       |               ^~~~
>       |               _b
>

**Comment by mertcandav** [comment]
Hi, thanks for your contribution. This bug belongs to expression builder.
Fixed with this commit: 3465083

**Comment by ghost** [comment]
I can confirm that it works after the fix, thanks :) Keep up the good work!
